### PR TITLE
Update text to reflect sample changes

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #7668 

Sample changes can be seen at https://github.com/dotnet/samples/pull/1664

### CODE REVIEW

File: docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md [hunk 1]
```diff
@@ -9,17 +9,21 @@ helpviewer_keywords:
...
-## Example  
- [!code-vb[VbVbalrMyComputer#25](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrMyComputer/VB/Class2.vb#25)]  
-  
-## Robust Programming  
- A <xref:System.ArgumentException> exception is raised if an application with the requested process identifier cannot be found.  
-  
-## .NET Framework Security  
- The call to the `Shell` function requires full trust (<xref:System.Security.SecurityException> class).  
+
+This example uses the `Shell` function to start the Notepad application and then prints a sentence by sending keystrokes using the `My.Computer.Keyboard.SendKeys` method.
```
Comment (Youssef1313) [inline docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md]: I think using `xref`s here would be better. I would also like to either use the word "function" or the word "method".

File: docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md [hunk 2]
```diff
@@ -9,17 +9,21 @@ helpviewer_keywords:
 ms.assetid: f1303184-fce4-44fb-88b4-aac5f42d5d77
 ---
 # How to: Start an Application and Send it Keystrokes (Visual Basic)
```
Comment (Youssef1313) [inline docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md]:
```suggestion
# How to: start an application and send it keystrokes (Visual Basic)
```

File: docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md [hunk 3]
```diff
@@ -9,17 +9,21 @@ helpviewer_keywords:
...
-## .NET Framework Security  
- The call to the `Shell` function requires full trust (<xref:System.Security.SecurityException> class).  
+
+This example uses the `Shell` function to start the Notepad application and then prints a sentence by sending keystrokes using the `My.Computer.Keyboard.SendKeys` method.
+
+## Example
+
+[!code-vb[VbVbalrMyComputer#25](~/samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbVbalrMyComputer/VB/Class2.vb#25)]
+
+## Robust Programming
```
Comment (Youssef1313) [inline docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md]:
```suggestion
## Robust programming
```

File: docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md [hunk 4]
```diff
@@ -9,17 +9,21 @@ helpviewer_keywords:
...
+
+A <xref:System.ArgumentException> exception is raised if an application with the requested process identifier cannot be found.
```
Comment (Youssef1313) [inline docs/visual-basic/developing-apps/programming/computer-resources/how-to-start-an-application-and-send-it-keystrokes.md]:
```suggestion
An <xref:System.ArgumentException> exception is raised if an application with the requested process identifier cannot be found.  
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Review by Thraka — APPROVED** [review]
LGTM. @Youssef1313 has some good quick suggestions to implement.

**Pull request merged**